src/coreclr/src/jit/gentree.h

```diff
@@ -4728,8 +4728,8 @@ struct GenTreeJitIntrinsic : public GenTreeOp
     ClassLayout* m_layout;
 
     union {
-        var_types      gtOtherBaseType; // For AVX2 Gather* intrinsics
-        regNumberSmall gtOtherReg;      // For intrinsics that return 2 registers
+        var_types gtAuxiliaryType; // For intrinsics than need another type (e.g. Avx2.Gather* or SIMD (by element))
```

Comment:
Why rename to `AuxiliaryType` but not `AuxiliaryReg`?

Reply:
Since I didn't touch gtOtherReg in other files. 

The intent of the renaming was to get rid of **Base**Type since for SIMD By Element intrinsics I use this field to encode a **SIMD** type of an indexed element while in some other case we do use it to keep the *"other"* base type (e.g. in case of wide/long intrinsics). I could've renamed it to gtOtherType.

Do you want me to rename the gtOtherReg -> gtAuxiliaryReg?

Reply:
Was mostly just interested. This makes sense, thanks!